__tests__/app.test.js

```diff
@@ -95,6 +95,38 @@ describe("GET /api",()=>{
             })
             
         })
+describe("GET /api/articles", ()=>{
+    test('200: responds with an array of article object', ()=>{
+        return request(app)
+        .get("/api/articles")
+        .expect(200)
+        .then(({body})=>{
+            const {articles} = body
```

Comment:
Need to test for this:

the articles should be sorted by date in descending order.

You can use jest sorted for this.